tests/phpunit/Compat/WC_Stripe_Email_Customer_Failed_Refund_Test.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+namespace WooCommerce\Stripe\Tests\Compat;
+
+use WC_Order;
+use WC_Stripe_Email_Customer_Failed_Refund;
+use WP_UnitTestCase;
+
+class WC_Stripe_Email_Customer_Failed_Refund_Test extends WP_UnitTestCase {
+	/**
+	 * Tests for the `__constructor` method.
+	 *
+	 * @return void
+	 */
+	public function test_instance() {
+		$email = new WC_Stripe_Email_Customer_Failed_Refund();
+
+		$this->assertInstanceOf( WC_Stripe_Email_Customer_Failed_Refund::class, $email );
+
+		$this->assertEquals( 'failed_refund_customer', $email->id );
+		$this->assertEquals( 'Sent to a customer when a refund request fails. The email contains the original order information.', $email->description );
+		$this->assertEquals( 'emails/failed-refund-customer.php', $email->template_html );
+		$this->assertEquals( 'emails/plain/failed-refund-customer.php', $email->template_plain );
+		$this->assertEquals( plugin_dir_path( WC_STRIPE_MAIN_FILE ) . 'templates/', $email->template_base );
+	}
+
+	/**
+	 * Tests for the `get_template_params` method.
+	 *
+	 * @return void
+	 */
+	public function test_get_template_params() {
+		$email = new WC_Stripe_Email_Customer_Failed_Refund();
+
+		$order = $this->getMockBuilder( WC_Order::class )
+			->disableOriginalConstructor()
+			->getMock();
+
+		$email->object = $order;
+
+		$params = $email->get_template_params();
+
+		$this->assertArrayHasKey( 'order', $params );
+		$this->assertArrayHasKey( 'reason', $params );
+		$this->assertArrayHasKey( 'email_heading', $params );
+		$this->assertArrayHasKey( 'sent_to_admin', $params );
+		$this->assertArrayHasKey( 'plain_text', $params );
+		$this->assertArrayHasKey( 'email', $params );
+
+		$this->assertEquals( $order, $params['order'] );
```

Comment:
Can we also test some of the basic fields to make sure they are pulling from the expected source values?

Reply:
Can you give me some examples of what you expect to be tested here? Do you mean like the target email address? If so, I can create another test for those. Here, I am testing the `get_template_params` method specifically.

Reply:
Sorry I wasn't more specific! I was wanting to make sure that the values actually are what we expect, and not just that the values are defined. The only value we are checking is `order`, and not any of the other fields.